web/src/starred_messages.js

```diff
@@ -82,5 +82,4 @@ export function rerender_ui() {
 
     stream_popover.hide_topic_popover();
     top_left_corner.update_starred_count(count);
-    stream_popover.hide_starred_messages_popover();
```

Comment:
Should we delete this since we do not have such function?

Reply:
The situation where we'd want to do something is that, if you have two browser windows open with the starred messages popover open, and enable displaying starred message counts in one ... does that leave the other incorrectly offering to enable it, when it's now already enabled? Hiding the popover here avoids potentially displaying that sort of stale state.

Reply:
Thanks for explanation! I'll fix this!